authorcontent.Rmd

```diff
@@ -21,8 +21,8 @@ Both are given the same [promotional treatment](#promote) by rOpenSci.
 
 ## Themes of posts {#postthemes}
 
-Read a few [blog posts and tech notes](https://ropensci.org/archive/) and consider what you like (or not) about them.
-Most posts written by community members are about packages that have passed rOpenSci software peer review. However, we have posts on a range of topics and we encourage you to consider these or to propose others.
+Read a few [posts](/archive/) and consider what you like (or not) about them.
```

Comment:
The relative link won't work from the blog guide subdomain.

Reply:
fixed

Reply:
it doesn't look fixed?